addons/crm_livechat/models/mail_channel.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import re
+from odoo import models, _
+
+
+class MailChannel(models.Model):
+    _inherit = 'mail.channel'
+
+    def _define_command_lead(self):
+        return {'help': _('Create a new lead')}
+
+    def _execute_command_lead(self, **kwargs):
+        key = kwargs.get('body').split()
+        partner = self.env.user.partner_id
+        msg = _('Something is missing or wrong in command')
+        channel_partners = self.env['mail.channel.partner'].search([('partner_id', '!=', partner.id), ('channel_id', '=', self.id)], limit=1)
+        if key[0].lower() == '/lead':
```

Comment:
@jem-odoo You are right! typing other command it will not execute.
But, What if user select /lead command and he/she write some word before /lead (like: test_word /lead) ?